workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/Model/Details/TypeDetails/UnityEnumDetails.cs

```diff
@@ -1,16 +1,35 @@
 using System.Collections.Generic;
 using System.Linq;
+using NLog;
 
 namespace Improbable.Gdk.CodeGeneration.Model.Details
 {
     public class UnityEnumDetails : GeneratorInputDetails
     {
         public readonly IReadOnlyList<(uint, string)> Values;
 
+        private static readonly Dictionary<string, uint> EnumMinimums = new Dictionary<string, uint>();
+
+        private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
+
         public UnityEnumDetails(string package, EnumDefinition rawEnumDefinition)
             : base(package, rawEnumDefinition)
         {
-            Values = rawEnumDefinition.Values.Select(value => (value.Value, value.Name)).ToList();
+            var min = rawEnumDefinition.Values.Select(list => list.Value).Min();
```

Comment:
Can we have empty schema enums? If so, I think `Min()` will throw if there are no elements in the sequence.

Reply:
No, schema compiler fails